pvlib/modelchain.py

```diff
@@ -1128,15 +1128,15 @@ def infer_temperature_model(self):
         elif {'noct', 'module_efficiency'} <= params:
             return self.noct_sam_temp
         else:
-            raise ValueError(f'could not infer temperature model from '
-                             f'ModelChain.system.  '
-                             f'If Arrays are used to construct the PVSystem, '
-                             f'check that all Arrays in '
-                             f'ModelChain.system.arrays '
-                             f'have parameers for the same temperature model. '
-                             f'If Arrays are not used, check that the PVSystem '
-                             f'attributes `racking_model` and `module_type` '
-                             f'are valid.')
+            raise ValueError('could not infer temperature model from '
```

Comment:
```suggestion
            raise ValueError('Could not infer temperature model from '
```

Reply:
@matsuobasho this capitalization change will also require the tests to be updated.  See the test failures here: https://github.com/pvlib/pvlib-python/actions/runs/8146219415/job/22264244062?pr=1977#step:9:134